docs/content/dependencies-file.md

```diff
@@ -41,6 +41,7 @@ Only direct dependencies should be listed and you can use the [`paket simplify`
 Paket supports the following source types:
 
 * [NuGet](nuget-dependencies.html)
+* [Git](git-dependencies.html)
```

Comment:
Actually the diff was confusing me. Double-checked both in my source and in the updated online docs and we're good. :100: